### DISCUSSION TITLE
What is the npm command that I can use to install this library.

### DISCUSSION THREAD

**Discussion by AndrewCromar** [faker-js/faker#71] [Q&A] [answered]
I'm trying to install this library in my project however when running "npm i faker" it installs the original version, what npm command can I use?

Thanks for any help! :)

**Comment by damienwebdev** [comment, marked as answer]
`npm install @faker-js/faker@{version}` will work, we have back-ported all of the old versions from the original npm registry.

**Reply by AndrewCromar** [reply]
Thanks!

**Comment by Shinigami92** [comment]
We should add a section in the `README.md`
